src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -649,6 +679,95 @@ def get_observation(self, event: Event) -> pd.DataFrame:
         for col in ["employer_address_id", "employer_name"]:
             df_w2[col] = df_w2["employer_id"].map(emp[col])
 
+        # re-initialize income-to-date series for next year of income counting
+        self.income_last_year = self.income_this_year
+        self.income_this_year = empty_income_series()
+
+        return df_w2[self.output_columns]
+
+
+class TaxDependentsObserver(BaseObserver):
+    """Class for observing columns relevant to identifying dependents in tax data.
+
+    This is most important for the 1040 data, but relies on data from
+    the W2 observer, and is better represented in a "long" form with a
+    row for each guardiant/dependent pair
```

Comment:
```suggestion
    row for each guardian/dependent pair
```

Reply:
good catch.  i will add it myself to avoid needing an additional merge on my side.